workloads/http.yml

```diff
@@ -0,0 +1,121 @@
+---
+#
+# Runs HTTP workload on an existing RHCOS cluster
+#
+
+- name: Runs HTTP workload on a RHCOS cluster
+  hosts: orchestration
+  gather_facts: true
+  remote_user: "{{orchestration_user}}"
+  vars_files:
+    - vars/http.yml
+  vars:
+    workload_job: "http"
+    enable_pbench_agents: true
+    workload_job_privileged: true
+    workload_job_node_selector: true
+    workload_job_taint: true
```

Comment:
Good catch! The env cleanup, is on my TODO.  As I said below, do we need to have these for every workload in `workloads/vars/<workload>.yml`?

Reply:
Well I think it would be convenient if all environmentally defined variables are provided in the vars file `workloads/vars/http.yml` with the exception of non-environmentally defined (or vars that we don't need provided from jenkins) could be defined at the top of the workload playbooks.  A good example of this is the test job which is running a sshd daemon in the workload job container, thus would always require the node selector to be placed on our workload/pbench node which has a public ip and can thus be reachable via sshd.

In regards to if every workload would require those vars (enable node selector/taint and privileged) to make the goal of supporting environments not provisioned by our install automation we will need to provide options to run without those items switched on.

Reply:
I've dropped most of these from the environment in `workloads/vars/http.yml` now, only `ENABLE_PBENCH_AGENTS` is left and removed `enable_pbench_agents` from `workloads/workloads/http.yml`.  `workload_job_privileged` is false now, so, we're only left with `workload_job_node_selector` and `workload_job_taint` set to true.  `workload_job_node_selector` can be confusing, as it is a `boolean` which forces a node with a  pbench label to be used.  Also not very happy about the `workload_job_taint` naming.  I will look into ways of making these false so that the variables don't need to be exposed at all.

Reply:
I will be playing with this pr more later today for more feedback.  My initial thoughts for those vars was that turning them off would be a tradeoff between workload isolation and ability to run the workloads on environments we didn't provision.  I hope that helps explain otherwise we can pair up over bluejeans sometime to discuss.  Also I am up for better names of course!

Reply:
Could be just my lack of understanding what's the purpose for having 2 variables.  From what I can see 1 variable could easily replace them.  Something like `run_on_pbench_node` but that's not the variable name I'm suggesting.  Variable naming is one of the the hardest problems in computer science ;) Let's discuss offline.